sympy/stats/__init__.py

```diff
@@ -34,6 +35,8 @@
 35/6
 >>> simplify(P(Z>1)) # Probability of Z being greater than 1
 1/2 - erf(sqrt(2)/2)/2
+>>> mean(X) # Average value of outcome of dice
```

Comment:
I would merge this with the E(X+y) as 
```python
E(X + Y)  # or mean(X + Y), the expected average of two die
```

Reply:
Yes, this can be done. But a specific example should also be present, I think

Reply:
Tell me if it is better this way